lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -342,6 +344,18 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
             .subscribe(pagSize => {
                 this.maxItems = this._pagination.maxItems = pagSize;
             });
+
+        this.uploadService.fileUploadComplete
```

Comment:
I think it would be better to have a `NotificationService.nodesDataChanged().subscribe(this.reload)`